config/config.go

```diff
@@ -186,9 +216,6 @@ func (cfg *AggregateStatsConfig) Validate() error {
 
 // ServerConfig contains the API server configuration.
 type ServerConfig struct {
-	// ChainID is the chain ID (normally latest) for the server API
-	ChainID string `koanf:"chain_id"`
```

Comment:
turns out this is still used in one of our API responses. keeping it for now. we can remove it later if we want

Reply:
actually let me try removing it in a separate commit

Reply:
removing that field. good luck to everyone downstream